tests/test_vector_sample.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+import unittest
+import time
+from pprint import pprint
+from flask.json import loads as json_load
+from flask.json import dumps as json_dump
+try:
+    from .test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
+except:
+    from test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
+
+
+__license__ = "GPLv3"
+__author__ = "Sören Gebbert, Markus Neteler"
```

Comment:
```suggestion
__author__ = "Markus Neteler"
```

This is a new file, do we need to add Sören here as author?